run_commands.txt

```diff
@@ -4,7 +4,6 @@ cpp/wordcount
 cpp/wordcount_clang
 cpp/wordcount_baseline
 d/wordcount
-elixir/wordcount
```

Comment:
I'll take look into the exact elixir/erlang issue, but fear that esl does
not provide 16.04 builds so far.

Sam Van Oort <notifications@github.com> schrieb am Fr., 1. Apr. 2016 16:37:

> In run_commands.txt
> <https://github.com/juditacs/wordcount/pull/70#discussion_r58214473>:
>
> > @@ -4,7 +4,6 @@ cpp/wordcount
> >  cpp/wordcount_clang
> >  cpp/wordcount_baseline
> >  d/wordcount
> > -elixir/wordcount
>
> Until I fix erlang installation, can't build/run it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/juditacs/wordcount/pull/70/files/2b468fcc9f3228086d6ee68fc30fb2e868514a99..b317f30048000aed710f3d4f46f201a18d8339b4#r58214473>
>